tests/vnc_test.go

```diff
@@ -155,6 +161,54 @@ var _ = Describe("[rfe_id:127][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			_, err = wrappedRoundTripper.RoundTrip(req)
 			Expect(err).ToNot(HaveOccurred())
 		})
+
+		It("should connect to vnc with --proxy-only flag", func() {
+			By("Invoking virtctl vnc with --proxy-only")
+			proxyOnlyCommand := tests.NewVirtctlCommand("vnc", "--proxy-only", "--namespace", vmi.Namespace, vmi.Name)
+
+			r, w, _ := os.Pipe()
+			os.Stdout = w
+
+			outC := make(chan string)
+			go func() {
+				var buf bytes.Buffer
+				io.Copy(&buf, r)
+				outC <- buf.String()
+			}()
```

Comment:
This block (lines 172 - 177) is not necessary, because the `proxyOnlyCommand` will be executed in a goroutine, so this method can simply read from `r`.